assembl/static2/js/app/utils/linkify.js

```diff
@@ -25,4 +25,9 @@ export function addProtocol(url: string): string {
   return hasProtocol ? url : `https://${url}`;
 }
 
+export function addIframeForMindManager(html: string): string {
+  const url = /(<a href="(https:\/\/share.mindmanager.com[^\s]+)".*<\/a>)/gi;
```

Comment:
Make sure this URL path does NOT change. Also, it is always https mode, correct?

Reply:
Should be (link format given by Kristy)